composeApp/src/androidMain/kotlin/com/sangyoon/parkingpass/presentation/ui/CameraScreen.android.kt

```
@@ -0,0 +1,197 @@
+                                try {
+                                    val image = currentController.captureImage()
+                                    if (image != null) {
+                                        // 이미지 분석 - 번호판이 인식되면 true 반환
+                                        val recognized = viewModel.recognizePlateFromImage(image)
+                                        if (recognized) {
+                                            // 번호판 인식 성공 - 카메라 화면 종료
+                                            onImageCaptured(image)
+                                            return@LaunchedEffect
+                                        }
+                                    }
+                                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)